mock_event.json

```diff
@@ -0,0 +1,186 @@
+{
+    "ref": "refs/heads/feature",
+    "before": "b40c4ce16063a5736bfd70f1789d8895e877fed1",
+    "after": "8bcdcf40ef2bde59cc907da5cf88a112b7c55de5",
+    "repository": {
+        "id": 596708517,
+        "node_id": "R_kgDOI5EMpQ",
+        "name": "test_pr",
+        "full_name": "syednabilashraf/test_pr",
+        "private": false,
```

Comment:
[mock_event.json, line 10]: The value of the session_token key in the config dictionary is not defined.